go.mod

```
@@ -0,0 +1,56 @@
+module github.com/wantedly/oauth2-proxy-manager
+
+go 1.19
```

Different from the Dockerfile version (1.20) 👀 
Is it as intended?

---

I missed it, thank you!

```suggestion
go 1.20
```